### PR TITLE
Glyphicon for #3411

### PR DESCRIPTION
**Pull request by tjwds**
…but it looks… blurry?  I'd rather use "icon-show-lines" but that set isn't in the bootstrap package used here.  

[capture decran 2014-05-04 a 1 10 21 pm](https://cloud.githubusercontent.com/assets/7093793/2873507/00db5a16-d3af-11e3-8dd2-1b9383661dc4.png)

[https://github.com/HabitRPG/habitrpg/issues/3411](https://github.com/HabitRPG/habitrpg/issues/3411)

### CODE REVIEW

**Comment by lefnire** [comment]
I don't think this icon looks quite right. Maybe `glyphicon glyphicon-align-justify` ([screen](http://gyazo.com/8d82c909bb05b56007df0a75f3ea5117))? Else we could just add our own icon.

Is it really called "trigram for heaven"? that's awesome.

**Comment by benmanley** [comment]
I'd prefer to keep the mighty Trigram by using a custom icon. There's a simple three bar icon in the [Entypo](http://www.entypo.com) set, for example (13 across, 7 down). Where should we put custom fonts though?

**Comment by tjwds** [comment]
Align justify looks great to me. I don't think adding a font would be that good for load time, to be honest.

Either away there'd probably need to be some styling tweaks to make it look good.

**Comment by benmanley** [comment]
I used the Trigram as the alternatives are quite heavy when scaled down. Here's align-justlfy:

justify

Plus three lines is more universally accepted as signifying "Menu".

One of the reasons I used the character code was because I was hesitating to add a new font. But FWIW, I’m not suggesting we add the whole of Entypo. It would be a custom font that we build up on an icon by icon basis. Since we're not using that many icons at the moment, we might even be able to move away from glyphicons altogether and only serve the icons we need.

**Comment by tjwds** [comment]
I got my hopes up for the moment for [Octicons](https://github.com/styleguide/css/7.0), but it's not free to use. :c 

I think if we [update bootstrap-glyphicons](https://github.com/marcoceppi/bootstrap-glyphicons), we'd be able to use icon-show-lines.

**Comment by tjwds** [comment]
Or maybe just keep the trigram and, if someone's on a device necessitating it, use one of the less pretty options?  Moving away from glyphicons sounds like a lot of work. 

Additionally, is there even attribution for the glyphicons somewhere on the site?

(doublepost lol sorry)

File: views/shared/header/menu.jade [hunk 1]
```diff
@@ -2,7 +2,7 @@ nav.toolbar(ng-controller='AuthCtrl')
   ul.toolbar-mobile-nav
     li.toolbar-mobile
       a(ng-click='expandMenu("mobile")', ng-class='{active: _expandedMenu=="mobile"}')
-        span  &#9776;
+        span.glyphicon.glyphicon-th-list
```
Comment (Alys) [inline views/shared/header/menu.jade:5]: This instance of `&#9776;` has been replaced by glyphicon-align-justify in https://github.com/HabitRPG/habitrpg/pull/4855

I'm not sure if the other instances below need to be replaced for the sake of mobile devices (except maybe tablets?). See discussion in https://github.com/HabitRPG/habitrpg/issues/3411

**Comment by Alys** [comment]
I'm closing this because it will conflict with a recent change, but we can keep discussing requirements in https://github.com/HabitRPG/habitrpg/issues/3411

**Pull request closed without merging**